src/main/webapp/app/exercises/quiz/shared/questions/drag-and-drop-question/drag-and-drop-question.component.ts

```diff
@@ -285,8 +300,22 @@ export class DragAndDropQuestionComponent implements OnChanges {
 
     /**
      * counts the amount of right mappings for a question by using the isLocationCorrect Method
+     * drop locations that are correctly left blank are excluded because they are excluded from grading as well
      */
     countCorrectMappings(): void {
         this.correctAnswer = this.question.dropLocations!.filter((dropLocation) => this.isLocationCorrect(dropLocation)).length;
     }
+
+    /**
+     * counts the amount of incorrect mappings for a question by using the isLocationCorrect Method
+     */
```

Comment:
I would stick to the documentation style already used in this component, i.e. "Count the amount ..."
-> actually there are multiple styles mixed already, so it's up to you which one to use or maybe unify it anyways :)